ui/app/src/components/FormsEngine/components/FieldInformationDialog.tsx

```
@@ -0,0 +1,119 @@
+						const value = field[basicPropsMap[descriptor.id] ?? descriptor.id];
+						const renderHtml = descriptor.id === 'help';
+						return (
+							<Grid container spacing={2} key={descriptor.id} sx={{ mt: 0.5 }}>
+								<Grid size={{ xs: 4 }}>
+									<Typography color="text.secondary">
+										{getPossibleTranslation(descriptor.name, formatMessage)}
+									</Typography>
+								</Grid>
+								<Grid size={{ xs: 8 }}>
+									{renderHtml ? (
+										<Typography dangerouslySetInnerHTML={{ __html: value ?? '-' }} />
```

Biome (2.3.14)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)